src/ripple/app/tx/impl/InvariantCheck.h

```diff
@@ -365,6 +365,32 @@ class NFTokenCountTracking
         beast::Journal const&);
 };
 
+/**
+ * @brief Invariant: Trust line balance cannot be negative after Clawback.
```

Comment:
Consider re-writing this. You mean something different from "trust line balance" here. `accountHolds` is the balance with respect to one side of the trust line. The balance on the trust line itself _can_ be negative.

Reply:
Thanks for the latest patch. I think this is ready to be approved, but I would like to re-write this comment first. Thanks!